great_expectations/rule_based_profiler/config/base.py

```diff
@@ -220,3 +238,6 @@ class Meta:
         values=fields.Nested(RuleConfigSchema, required=True),
         required=True,
     )
+
+
+ruleBasedProfilerConfigSchema = RuleBasedProfilerConfigSchema()
```

Comment:
@cdkini Let us add the same for all the components that have a `Schema` representation.  Thanks.

Reply:
done!